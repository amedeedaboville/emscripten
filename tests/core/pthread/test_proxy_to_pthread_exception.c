#include <emscripten.h>

int main() {
  EM_ASM(missingFunc());
  return 0;
}
