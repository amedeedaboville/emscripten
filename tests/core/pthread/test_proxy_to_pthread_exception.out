ReferenceError: missingFunc is not defined
